.github/workflows/release_linux.yml

```diff
@@ -0,0 +1,69 @@
+name: LinuxRelease
+
+on:
+  push:
+    branches:
+      - rel-*
+  # test this PR for now, remove it before the merge
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
+        architecture: ['x64']
+    steps:
+    - uses: actions/checkout@v2
+    - name: Checkout submodules
+      shell: bash
+      run: |
+        auth_header="$(git config --local --get http.https://github.com/.extraheader)"
+        git submodule sync --recursive
+        git -c "http.extraheader=$auth_header" -c protocol.version=2 submodule update --init --force --recursive --depth=1
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+        architecture: ${{ matrix.architecture }}
+    - name: Install python dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install pytest==5.4.3 nbval numpy
+        python -m pip install virtualenv
+        python -m virtualenv py$(python.version)
+        source py$(python.version)/bin/activate
+        python -m pip install numpy protobuf==3.11.3
+        sudo apt-get install protobuf-compiler libprotoc-dev
+
+    - name: Build manylinux Python wheels
+      env:
+        ONNX_BUILD_TESTS: 1
+        USE_MSVC_STATIC_RUNTIME: 1
+        ONNX_ML: 1
+        CMAKE_ARGS: "-DONNX_USE_LITE_PROTO=ON"
+      uses: RalfG/python-wheels-manylinux-build@v0.3.3-manylinux2010_x86_64
+      with:
+        python-versions: 'cp36-cp36m cp37-cp37m cp38-cp38m'
+        build-requirements: 'numpy protobuf'
```

Comment:
You may consider to install protobuf python package from source. Otherwise you should be careful about the versioning. 
Here you will get the latest protobuf python package, in most of the cases, it's fine. Unless you got a newer version of protoc and libprotobuf C/C++ library.

Reply:
Yes... Some of these libraries are not the latest ones and they need to be updated manually...